perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AttributesManagerMethod.java

```diff
@@ -3974,6 +3975,25 @@ public Void call(ApiCaller ac, Deserializer parms) throws AttributeAlreadyMarked
 		}
 	},
 
+	/*#
+	 * Converts attribute to nonunique - unmarks unique flag from attribute definition, and deletes all values
+	 * from a special table with unique constraint that ensures that all values remain unique.
+	 * Already nonunique attribute is skipped.
+	 *
+	 * @param attrDefId int AttributeDefinition <code>id</code>
+	 * @throw PrivilegeException Insufficient permissions.
```

Comment:
We do not mention ``PriviligeException`` in RPC docs, since its automatically added by docs parsing too.

Reply:
Removed.